tests/ttnn/integration_tests/whisper/test_ttnn_functional_whisper.py

```diff
@@ -230,13 +226,12 @@ def test_decoder_layer(device, ttnn_model, model_name, batch_size, sequence_size
     )
     output = ttnn.to_torch(output)
 
-    assert_with_pcc(torch_output, output, 0.97)
+    assert_with_pcc(torch_output, output, 0.99)
 
 
-@pytest.mark.skipif(is_wormhole_b0() or is_blackhole(), reason="Unsupported on WH and BH")
```

Comment:
should the skip for blackhole be kept here (if it has not been verified to work)? same comment for other skips